### ISSUE TITLE
I think this section is out of date?

### ISSUE THREAD

**Issue by kdkorthauer** [mtello22/KorthauerLab#6]
Also seems to refer to plots that are not included. 

https://github.com/mtello22/KorthauerLab/blob/3df8ec2c497323b50ef302e0ee84beee70f23dbf/VMRassociation/SupervisedLearning/Reports/CommitteeReport/CommitteeReport.Rmd#L419

**Comment by mtello22** [comment]
Yes, it is. I updated the cell-type plot and removed the first set of violin plots. VMRs no longer show a bimodal-like shape and look more similar to promoter results. It is likely that modifying the parameters for VMR definition changed that.

**Comment by kdkorthauer** [comment]
Ok; it looks like these results didn't change (still no better than random guess), so I’d suggest to explore training the models on a summary of that metric (indicate for future work for the purposes of this report).

**Comment by mtello22** [comment]
What do you mean by a summary of that metric?
Currently, I have also results for a different MAD threshold and a different feature table version of each MAD threshold with a different encoding of TF variables. However, the results of the basic regression metrics look very similar so I did not proceed with further analyses.

**Comment by kdkorthauer** [comment]
average or median

**Comment by kdkorthauer** [comment]
Is the plot in Fig 7 the binary metric (within gene AND cell type, whether the ranking of the two samples is correct)? Should also add back in the converse configuration (within gene and sample, the spearman correlation amongst the cell types) - I think this is more relevant for the way the VMRs are defined to pick up cell type variability. This is the one I'm suggesting to use an average or median over all genes as the training metric

**Comment by mtello22** [comment]
I would like to discuss this part a bit more in our next one-one, I understand that using the average of each gene across all samples would be a better metric rather than using every value in the violin plot. Is that correct?

**Comment by kdkorthauer** [comment]
I'm not just talking about the violin plot, I'm talking about using that as THE metric to tune and fit models.

**Comment by mtello22** [comment]
Ok, I made the implementation yesterday. The models are running. 
I just added the clarification in the report

**Comment by kdkorthauer** [comment]
The discussion in the report of the alternate metric and how it will be used should be more specific. The sentence "We will also perform different ways of summarizing the information presented for each gene across all samples" is particularly vague - what ways?  Also doesn't indicate that you will be exploring **training** on the per gene metric.
